packages/app/pages/api/models/[modelName]/documents/index.ts

```diff
@@ -44,15 +45,20 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
                 )
             }
 
-            const parsedDocument = JSON.parse(req.body)
-            const [error, data] = await createDocument(
+            const [parsingError, parsedDocument] = safeParseJSON(req.body)
```

Comment:
Allows something like cURL to upload documents via `-d @filename.json`, which is pretty useful when working locally (and might be useful for `meditor-api-scripts`). See tests for more complete description.